extensions/amp-analytics/analytics-vars.md

```diff
@@ -0,0 +1,105 @@
+    ...
+  },
+  "vars": {
+    "account": "ABC123"
+  },
+  "triggers": {
+    ...
+  }
+}
+</script>
+</amp-analytics>
+```
+
+## Supported Vars
+
+**random** provides a random value every time a request is being constructed.
+
+Example value: `0.12345632345`
+
+**canonicalUrl** provides the canonical URL of the current document.
+
+Example value: `http%3A%2F%2Fexample.com%3A8000%2Fanalytics.html`
+
+**canonicalHost** provides the canonical URL's host.
+
+Example value: `example.com`
+
+**canonicalPath** provides the canonical URL's path part.
+
+Example value: `%2Fanalytics.html`
+
+**clientId** provides a per document-source-origin (The origin of the website where you publish your AMP doc) and user identifier. The client id will be the same for the same user if they visit again within one year. If the AMP document is not served through the AMP CDN, the client id will be replaced with a cookie of the name of the cid scope below. If the cookie is not present, an empty string will be returned.
+
+Please see below the required and optional arguments you may pass into `clientId` like a function. (spaces between arguments and values are not allowed)
+
+arguments:
+
+`cid-scope` (Required) - Name of the fallback cookie when the document is not served by an AMP proxy.
+`amp-user-notification` id (Optional) - Optionally make the clientId substitution be dependent on the dismissal of a user notification shown to the visitor of the page.
+
+Example usage: `${clientId(foo)}`
+
+Example value: `U6XEpUs3yaeQyR2DKATQH1pTZ6kg140fvuLbtl5nynbUWtIodJxP5TEIYBic4qcV`
+
+
+**documentReferrer** provides the referrer where the user came from. It is read from `document.referrer`. The value is empty for direct visitors.
+
+Example value: `https://www.google.com`
+
+**title** provides the title of the current document.
+
+Example value: `Example Pageview`
+
+**ampdocUrl** provides the AMP document's URL. The url contains the scheme, domain, port and full path. It does not contian the fragment part of the URL.
+
+Example value: `http://example.com:8000/examples.build/analytics.amp.max.html`
+
+**ampdocHost** provides the AMP document's URL host.
+
+Example value: `example.com`
+
+**pageViewId** provides a string that is intended to be random and likely to be unique per URL, user and day.
+
+Example value: `978`
+
+**timestamp** provides the number of seconds that have elapsed since 1970. (Epoch time)
+
+Example value: `1452710304312`
+
+**timezone** provides the user's time-zone offset from UTC, in minutes.
+
+Example value: `480` for pacific time.
+
+**scrollTop** provides the number of pixels that the user has scrolled from top. The value is generated by `viewport.getScrollTop()`.
+
+Example value: `0`
+
+**scrollLeft** provides the number of pixels that the user has scrolled from left. The value is generated using `viewport.getScrollLeft()`.
+
+Example value: `100`
+
+**scrollHeight** provides the total size of the page in pixels.
```

Comment:
Removed others. It turns out the viewport.* api I was talking about was AMP specific and does not make much sense here.